docker/README.md

```diff
@@ -37,18 +48,27 @@ containers to communicate with each other - it prevents worker to connect to Web
 
 on host machine.
 
-## Create directory structure
+**Create directory structure**
+
+In case you want to have the big files (isos and disk images, test and needles) outside of the docker volume container,
+you should create this file structrure from within the directory you are going to execute the container.
 
     mkdir -p data/factory/{iso,hdd} data/tests
 
-It is also necessary to either run all containers in privileged mode, or set selinux properly:
+It could be necessary to either run all containers in privileged mode, or set selinux properly. If you are having problems with it, run this command:
 
     chcon -Rt svirt_sandbox_file_t data
 
 ## Run the Data & WebUI containers
 
+**All data within the volume data container**
+
+    # openSUSE
+    docker run -d -h openqa_data --name openqa_data -v `pwd`/data/factory:/data/factory -v `pwd`/data/tests:/data/tests slindomansilla/openqa_data
+    docker run -d -h openqa_webui --name openqa_webui --volumes-from openqa_data -p 80:80 -p 443:443 slindomansilla/openqa_webui
```

Comment:
No, these are completely different images. The images under fedoraqa are using as operating system Fedora. the ones under slindomansilla are using openSUSE.

It could be, that an organisation openqa is created on docker hub, and then we could save the images on repositories opensuse and fedora. But I am not the maintainer of the fedora images. I don't have the dockerfile to build them and the ones under fedoraqa don't work.

Having the opensuse images under fedoraqa (domain for which I don't have push permission) would also not be good organized.

It is like the documentation for openQA when installing on openSUSE Leap, openSUSE Tumbleweed or Fedora. You have to duplicate the line "zypper in..." if you want to include all the flavors.